src/routes/game/GameView.vue

```diff
@@ -473,6 +473,11 @@ export default {
     isSpectating() {
       return this.gameStore.isSpectating;
     },
+    spectatingWrapperStyle() {
+      return {
+        zIndex: this.isSpectating ? 2401 : 3 // Allows spectators to access game menu wrapper in any moment
```

Comment:
Why this specic value for the high z-index? Did you try others?

Reply:
It's because the overlay has a z-index of 2400. Well, except that I checked it again and it's actually 2000 😄. So 2001 or higher is enough for the menu to be clickable. I'll change it.

Not sure how that z-index of 2000 is set in the "#waiting-for-opponent-counter-scrim" container though.